test/integration/org/org_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/gcloud"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/tft"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestOrg(t *testing.T) {
+
+	org := tft.NewTFBlueprintTest(t,
+		tft.WithTFDir("../../../1-org/envs/shared"),
+	)
+
+	org.DefineVerify(
+		func(assert *assert.Assertions) {
+			// perform default verification ensuring Terraform reports no additional changes on an applied blueprint
+			org.DefaultVerify(assert)
+
+			auditLogProjectID := org.GetStringOutput("org_audit_logs_project_id")
+			op1 := gcloud.Run(t, fmt.Sprintf("projects describe %s", auditLogProjectID))
+			assert.True(op1.Exists(), "project %s does not exist", auditLogProjectID)
```

Comment:
I think you are checking if the project exists? We can also do this in a loop since the gcloud and assert are common for all projects.
```suggestion
			assert.True(op1.Exists(), "project %s does exist", auditLogProjectID)
```

Reply:
fixed